packages/list-group/src/ListGroup.stories.tsx

```diff
@@ -20,16 +20,16 @@ export default {
     cards: false,
     selectable: false,
   },
-} as Meta;
+};
 
-export const Default: Story = ({ cards, selectable }) => (
-  <ListGroup cards={cards} selectable={selectable}>
-    <ListGroupItem>item</ListGroupItem>
-    <ListGroupItem>item</ListGroupItem>
-    <ListGroupItem>item</ListGroupItem>
-  </ListGroup>
-);
+export const _Default: StoryObj<typeof ListGroup> = {
```

Comment:
Can you check the difference between the props shown by default vs what's listed in additional props? the latest version of storybook is better at detecting props so if there's no difference we should remove the additional props page

Reply:
There are different props in the additional.
Default Props: 
<img width="1077" alt="Screenshot 2023-10-19 at 1 59 22 PM" src="https://github.com/Availity/availity-react/assets/147724065/3903454f-ccd1-4367-ba64-85c3a183e618">


Additional Prop:s
<img width="1108" alt="Screenshot 2023-10-19 at 1 58 03 PM" src="https://github.com/Availity/availity-react/assets/147724065/f37c016d-8ffd-42de-949e-efc52b3793ac">